### DISCUSSION TITLE
Welcome to PAPPL Discussions

### DISCUSSION THREAD

**Discussion by michaelrsweet** [michaelrsweet/pappl#100] [General]
## Welcome

I'm using Discussions as a place to connect with members of the PAPPL community. I hope that you:

* Ask questions you’re wondering about.
* Share ideas.
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community we build together.

To get started, comment below with an introduction of yourself and how you are using PAPPL.

**Comment by subnut** [comment]
The README says -

> PAPPL requires a POSIX-compliant host operating system such as Linux®, macOS®, QNX®, or VxWorks®, a "make" utility that supports the include directive

[POSIX](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/make.html) specifies that `make` shall support `include` directives.

I am curious why you thought about specifically mentioning that the `make` utility must support `include`. Have you seen any `make` utilites that _don't_ support `include` directives?

**Reply by michaelrsweet** [reply]
@Subnut Yes, unfortunately. BSD make is the most visible example (it uses "sinclude", "-include", and/or ".include", but not just "include"), as well as several older commercial UNIX implementations and Microsoft's nmake.
